lib/tabry/cli/all_in_one.rb

```diff
@@ -60,6 +60,11 @@ def self.define_completion_methods(cli_class, config, cmd_name: nil)
             require_relative "../shells/bash/wrapper"
             Tabry::Bash::Wrapper.run(args.cmd_line, args.comp_point, config: config)
           end
+
+          define_method :completion_fish do
+            require_relative "../shells/fish/wrapper"
+            Tabry::Fish::Wrapper.run(args.cmd_line, args.comp_point, config: config)
+          end
```

Comment:
Hmm, yeah, I see... I really don't like an extra top-level subcommand that will show up in all tabry commands' help and tab completion now, yet are not meant to be used directly. There are a couple different solutions (like making subcommands or flags for the fish completion), but the easiest is probably just an ENV var that can be checked here in this file.